android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
While updating the plugin version you should also update the gradle version. According to [this](https://developer.android.com/studio/releases/gradle-plugin), we need `gradle-v6.1.1` but we are using `v5.6.1`.

Reply:
Yeah, sorry about that, I missed that one. I have updated it now. Please have a look.